.github/workflows/autobuild.yml

```diff
@@ -412,34 +415,15 @@ jobs:
         with:
           product-path:             deploy/${{ steps.get-artifacts.outputs.artifact_1 }}
 
-      - name:                       Upload Artifact 1 to Release
-        # Every build job has at least one artifact. Therefore, no `if artifact_1` condition here.
-        # If the artifact is missing, this should fail.
+      - name:                       Deploy Artifacts to Release
         if:                         needs.create_release.outputs.publish_to_release == 'true'
-        id:                         upload-release-asset1
-        uses:                       actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url:               ${{ needs.create_release.outputs.upload_url }}
-          asset_path:               deploy/${{ steps.get-artifacts.outputs.artifact_1 }}
-          asset_name:               ${{ steps.get-artifacts.outputs.artifact_1 }}
-          asset_content_type:       application/octet-stream
-
-      - name:                       Upload Artifact 2 to Release
-        if:                         >-
-                                    steps.get-artifacts.outputs.artifact_2 != '' &&
-                                    needs.create_release.outputs.publish_to_release == 'true'
-        id:                         upload-release-asset2
-        uses:                       actions/upload-release-asset@v1
+        id:                         upload-release-assets
+        run:                        |
+          gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" "deploy/${{ steps.get-artifacts.outputs.artifact_2 }}" --clobber || gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" --clobber
```

Comment:
https://www.baeldung.com/yaml-multi-line#folded-style
This might be nicer as
```yaml
  run: >
    gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" "deploy/${{ steps.get-artifacts.outputs.artifact_2 }}" --clobber
    ||
    gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" --clobber
```
(I think that'll fold right....)